src/component/form/TextInputElement/CharCount.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import * as styles from './CharCount.scss';
+
+interface CharCount {
+    count: number;
+    parts: number;
+}
+
+// prettier-ignore
+const CharCount: React.SFC<CharCount> = ({ count, parts }) => (
+    <div className={styles.count}>
+        <div className={`${styles.tooltip}`}>
+            {count}/{parts}
+            <span className={styles.tooltipText}>
+                The character-counter provides an estimate of the number of messages your response
+                is expected to be split into by carriers. For example, the count <b>{count}/{parts}</b>
+                {' '}indicates that the current message will be split into <b>{parts}</b> messages and you have{' '}
+                <b>{count}</b> characters left until it will be split into <b data-spec="parts-next">{parts + 1}</b> messages.
+            </span>
```

Comment:
Careful in your new blurb to deal with language difference for plurality, missing here.